src/customer/strategies/masterpass-customer-strategy.ts

```diff
@@ -55,6 +55,8 @@ export default class MasterpassCustomerStrategy extends CustomerStrategy {
                     currency: cart.currency.code,
                     cartId: cart.id,
                     suppressShippingAddress: false,
+                    callbackUrl: window.location.origin +
+                        '/checkout.php?action=set_external_checkout&provider=masterpass&gateway=stripe&origin=checkout',
```

Comment:
Can this string be moved to a constant and be interpolated here vs. concatenated?